ballerina/client.bal

```diff
@@ -1,15 +1,993 @@
+        string resourcePath = string `/browse/featured-playlists`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    resource isolated function get browse/new\-releases(map<string|string[]> headers = {}, *GetNewReleasesQueries queries) returns inline_response_200_11|error {
+        string resourcePath = string `/browse/new-releases`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # Get Several Chapters
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - A set of chapters 
+    resource isolated function get chapters(map<string|string[]> headers = {}, *GetSeveralChaptersQueries queries) returns inline_response_200_6|error {
+        string resourcePath = string `/chapters`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # Get a Chapter
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - A Chapter 
+    resource isolated function get chapters/[string id](map<string|string[]> headers = {}, *GetAChapterQueries queries) returns ChapterObject|error {
+        string resourcePath = string `/chapters/${getEncodedUri(id)}`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # Get Several Episodes
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - A set of episodes 
+    resource isolated function get episodes(map<string|string[]> headers = {}, *GetMultipleEpisodesQueries queries) returns inline_response_200_4|error {
+        string resourcePath = string `/episodes`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # Get Episode
+    #
+    # + headers - Headers to be sent with the request 
+    # + queries - Queries to be sent with the request 
+    # + return - An episode 
+    resource isolated function get episodes/[string id](map<string|string[]> headers = {}, *GetAnEpisodeQueries queries) returns EpisodeObject|error {
+        string resourcePath = string `/episodes/${getEncodedUri(id)}`;
+        resourcePath = resourcePath + check getPathForQueryParam(queries);
+        return self.clientEp->get(resourcePath, headers);
+    }
+
+    # Get Available Markets
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - A markets object with an array of country codes 
+    resource isolated function get markets(map<string|string[]> headers = {}) returns inline_response_200_16|error {
```

Comment:
@Sachindu-Nethmin we don't prefer having these `inline_response_*` record types as the response type in connector APIs. This happens when the particular reponse type is defined as an inline schema in the OpenAPI contract. 

So in order to generated the response type with a proper name, you can update the OpenAPI spec file by moving the particular inline schema to the `components` section. 
Also Please make sure to mention record these improvements in the `sanitations.md` file.